core-web/libs/ui/src/lib/dot-container-options/dot-container-options.directive.ts

```diff
@@ -43,6 +43,7 @@ export class DotContainerOptionsDirective implements OnInit, OnDestroy {
             this.control.optionLabel = DEFAULT_LABEL_NAME_INDEX;
             this.control.optionValue = DEFAULT_VALUE_NAME_INDEX;
             this.control.optionDisabled = 'inactive';
+            this.control.filterBy = 'value.friendlyName,value.title';
```

Comment:
is this needed?

Reply:
Yes, there was a case where a container would come back from the api because it contained the query string in the title but not the friendly name. In that case p-dropdown would not display that container without this change.